resources/views/partials/forms/edit/category-select.blade.php

```diff
@@ -9,8 +9,6 @@
                 <option value="{{ $category_id }}" selected="selected" role="option" aria-selected="true"  role="option">
                     {{ (\App\Models\Category::find($category_id)) ? \App\Models\Category::find($category_id)->name : '' }}
                 </option>
-            @else
```

Comment:
Why was this removed?

Reply:
These partials are used elsewhere, and in places where a multi-select doesn't make sense

Reply:
So, looking back to where I was earlier, and it appears that in most locations such as the hardware edit blade, we only allow users to choose one value, and only one gets passed to the partial. Its on the custom report blade that we allow an array to pass.

Reply:
This is the fix discussed in slack, so if we want to isolate all the changes to the custom report blade specifically, we'd need to come up with another solution i think.

Reply:
as we went over in the stand up changes have been made. Not saying its complete, but I'll resolve this convo and we can open a new one for the edits